apps/web/components/eventtype/EventTypeDescriptionSafeHTML.tsx

```diff
@@ -1,12 +1,11 @@
 export type EventTypeDescriptionSafeProps = {
-  eventType: { description: string | null };
+  eventType: { description: string | null; descriptionAsSafeHTML: string | null };
 };
 
 export const EventTypeDescriptionSafeHTML = ({ eventType }: EventTypeDescriptionSafeProps) => {
   const props: JSX.IntrinsicElements["div"] = { suppressHydrationWarning: true };
   if (eventType.description)
-    // @ts-expect-error: @see packages/prisma/middleware/eventTypeDescriptionParseAndSanitize.ts
```

Comment:
cc @CarinaWolli Removed this now, as descriptionAsSafeHTML is provided from every place in your PR.